internal/services/lb/helpers_lb.go

```diff
@@ -56,10 +56,10 @@ func IsPrivateNetworkEqual(a, b *lbSDK.PrivateNetwork) bool {
 	if a.PrivateNetworkID != b.PrivateNetworkID {
 		return false
 	}
-	if !reflect.DeepEqual(a.DHCPConfig, b.DHCPConfig) {
+	if !reflect.DeepEqual(a.DHCPConfig, b.DHCPConfig) { //nolint:staticcheck
 		return false
 	}
-	if !reflect.DeepEqual(a.StaticConfig, b.StaticConfig) {
+	if !reflect.DeepEqual(a.StaticConfig, b.StaticConfig) { //nolint:staticcheck
```

Comment:
We need to add test and documentation to ensure that we detect when this field is removed from API responses and that the user are warned that the field is deprecated.